crates/bevy_time/src/fixed_timestep.rs

```diff
@@ -224,6 +224,15 @@ impl System for FixedTimestep {
     fn check_change_tick(&mut self, change_tick: u32) {
         self.internal_system.check_change_tick(change_tick);
     }
+
+    fn get_last_change_tick(&self) -> u32 {
```

Comment:
Same here. Why can't we just pipe through the internal system's last_change_tick?